steps/dns/steps.go

```diff
@@ -64,7 +69,18 @@ func (s Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioConte
 		if !ok {
 			return fmt.Errorf("invalid qtype '%s': permitted values '%s'", qtype, reflect.ValueOf(QueryTypes).MapKeys())
 		}
-		return session.SendQuery(ctx, qt, qname, recursive)
+		switch session.Transport {
+		case "UDP":
+			return session.SendQuery(ctx, qt, qname, recursive)
```

Comment:
For consistency, it's better to rename SendQuery to SendUDPQuery

Reply:
Done